crates/wascc-provider/src/lib.rs

```
@@ -33,10 +33,14 @@ pub struct WasccProvider {}
 #[async_trait::async_trait]
 impl Provider for WasccProvider {
     async fn init(&self) -> Result<(), failure::Error> {
-        let data = NativeCapability::from_file(HTTP_LIB)
-            .map_err(|e| format_err!("Failed to read HTTP capability {}: {}", HTTP_LIB, e))?;
-        host::add_native_capability(data)
-            .map_err(|e| format_err!("Failed to load HTTP capability: {}", e))
+        tokio::task::spawn_blocking(|| {
+            let data = NativeCapability::from_file(HTTP_LIB)
+                .map_err(|e| format_err!("Failed to read HTTP capability {}: {}", HTTP_LIB, e))?;
+            host::add_native_capability(data)
+                .map_err(|e| format_err!("Failed to load HTTP capability: {}", e))
+        })
+        .await
+        .expect("Could not spawn worker for initializing WasccProvider")
```

This should probably just return the error and let the caller handle it